GiniVisionExample/GiniVisionExample/ViewControllers/WebViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  WebViewController.swift
+//  GiniVisionExample
+//
+//  Created by Enrique del Pozo Gómez on 11/22/17.
+//  Copyright © 2017 Gini. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+final class WebViewController: UIViewController {
```

Comment:
In the past, I've had problems with Apple reviews regarding using in-app web views. They suggested using their implementation so that users can see the address bar and make sure there are no SSL warnings and stuff. I'm not sure that also applies for this case (probably not), but it might be worth considering if such a solution would improve the UX. I'd think about it if I was you.

Reply:
I have never heard about that, I know that Apple use to care about the JavaScript executed in that web view. In our case there is no plain since they are only plain HTMLs.

```
Note
In apps that run in iOS 8 and later, use the WKWebView class instead of using UIWebView. Additionally, consider setting the WKPreferences property javaScriptEnabled to false if you render files that are not supposed to run JavaScript.
```